llvm/test/DebugInfo/X86/template-as-type-param.ll

```diff
@@ -0,0 +1,124 @@
+; RUN: llc  -mtriple=x86_64-unknown-linux-gnu %s -o %t -filetype=obj
+; RUN:llvm-dwarfdump %t | FileCheck %s
```

Comment:
I'd recommend adding spaces to make this look more like the other tests.  So change to "RUN: " and "; CHECK:" for example.